autoload/mundo.py

```diff
@@ -262,7 +262,8 @@ def MundoMove(direction,move_count=1,relative=True,write=False):
     else:
         vim.command("call cursor(0, %d + 1)" % idx3)
 
-    if vim.eval('g:mundo_auto_preview') == '1':
+    if (vim.eval('g:mundo_auto_preview') != '0' and
+            int(vim.eval("get(g:, 'mundo_auto_preview_delay', 0)")) <= 0):
```

Comment:
(see comments below) If we go with `mundo_auto_preview_delay` as a non-zero value, there is no need for this check to see if it is set here.